src/bms_main.cpp

```diff
@@ -30,8 +30,10 @@ void bms_setup() {
   Serial.begin(115200);
 
   // The B line idle is 0
-  digitalWrite(TX_INVERSE_OUT_PIN, 0);
-  pinMode(TX_INVERSE_OUT_PIN, OUTPUT);
+  if (!Settings->board_locked) { // prevent it from sending packets along?
```

Comment:
Instead of this, just don't execute the line above, "Serial.begin()" if the board is locked.

Reply:
I take this comment back, not calling Serial.begin() would make UI look off when the user logs into the board to unlock it. Instead, modify the write lambda function passed into BmsRelay object creation so that it looks like:

[](uint8_t b) { !Settings.board_locked && Serial.write(b);}